docs/ww3_doxy_tmpl.md

```diff
@@ -0,0 +1,63 @@
+# WAVEWATCH III Doxygen Header Reference
+<br>
+
+_Doxygen markup headers to be placed directly above the respective Fortran code units._
+
+### FILE
+```
+!> @file  <file name>
+!> @brief  <one sentence description>
+!> 
+!> @author <author name> @date <dd-Mon-yyyy>
+```
+
+### MODULE
+```
+!> @brief <one sentence description>   # contents of _1. Purpose_ from original header
+!> 
+!> @details <extended description>
+!>
+!> @param <module var 1>               # ALL module variables documented
+!> ...
+!> @param <module var N>
+!> @author <author name> @date <dd-Mon-yyyy>
+```
+
+### PROGRAM
+```
+!> @brief  <one sentence description>      # contents of _1. Purpose_ from original header
+!>
+!> @details <extended description>          # contents of _2. Method_ from original header
+!> 
+!> @author <author name>  @date <dd-Mon-yyyy>
+```
+
+### SUBROUTINE
+```
+!> @brief  <one sentence description>       # contents of _1. Purpose_ from original header
```

Comment:
The brief tag is not needed for subroutines/functions. THe first sentence will be automatically used as the brief description.

Brief tags are required at the module and file level - autobrief does not work for files and modules, just subroutines and functions.